chapters/es/slides/chapter1_03_rule-based-matching.md

```diff
@@ -172,64 +172,64 @@ Estamos buscando cinco tokens:
 
 Un token compuesto únicamente por dígitos.
 
-Tres tokens insensibles a mayúsculas/minúsculas para "fifa", "world" y "cup".
+Tres tokens insensibles a mayúsculas/minúsculas para "copa", "mundial" y "fifa".
 
 Un token que está compuesto por puntuación.
 
-El patrón encuentra los tokens "2018 FIFA World Cup:".
+El patrón encuentra los tokens "2014 Copa Mundial FIFA:".
 
 ---
 
 # Encontrando por otros atributos del token
 
 ```python
 pattern = [
-    {"LEMMA": "love", "POS": "VERB"},
-    {"POS": "NOUN"}
+    {"LEMMA": "gustar", "POS": "VERB"},
+    {"POS": "VERB"}
 ]
 ```
 
 ```python
-doc = nlp("I loved dogs but now I love cats more.")
+doc = nlp("Me gustaba correr pero ahora me gusta nadar.")
 ```
 
 ```out
-loved dogs
-love cats
+gustaba correr
+gusta nadar
 ```
 
 Note: En este ejemplo estamos buscando dos tokens:
```

Comment:
This was already wrong before and based on mistakes in the original course – but I think it causes the notes not to show up, because the keyword is wrong.

```suggestion
Note: En este ejemplo estamos buscando dos tokens:
```